src/data/polynomial/degree/trailing_degree.lean

```diff
@@ -1,4 +1,8 @@
-import tactic
+/-
+Copyright (c) 2020 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
 import data.polynomial.degree.basic
 
```

Comment:
Ugh, this shouldn't have been merged without a module doc. @adomani when you have time, do you think you could make a new PR that adds a module doc explaining what is in this file, as described [here](https://leanprover-community.github.io/contribute/doc.html#header-comment)? This is what appears at the tops of the [doc pages](https://leanprover-community.github.io/mathlib_docs/analysis/mean_inequalities.html).

Reply:
I added the Copyright statement and a description of the module.

By the way, I added myself as the sole author, but all that I did was little more than copy another file and use a couple of times find/replace to make the proof work.  How can I credit the authors of the initial file?